tools/StaticAnalysis/Exceptions/Az.Peering/ExtraAssemblies.csv

```diff
@@ -0,0 +1,901 @@
+"Directory","AssemblyName","Severity","ProblemId","Description","Remediation"
+"E:\Interconnect\azure-powershell-pr\artifacts/Debug\Az.Accounts","Microsoft.ApplicationInsights","2","2000","Assembly Microsoft.ApplicationInsights is not referenced from any cmdlets assembly","Remove assembly Microsoft.ApplicationInsights from the project and regenerate the Wix file"
```

Comment:
@rileymckenna any reason that all other modules' contents were added to this file? You only need to add the contents corresponding to your module.

Reply:
nope i just copied it and pasted it to the directory per the troubleshooting guide. I updated it accordingly.